src/util/find_uid.c

```diff
@@ -86,18 +86,17 @@ static errno_t get_uid_from_pid(const pid_t pid, uid_t *uid)
         error = errno;
         if (error == ENOENT) {
             DEBUG(SSSDBG_TRACE_LIBS,
-                  "Proc file [%s] is not available anymore, continuing.\n",
+                  "Proc file [%s] is not available anymore.\n",
                       path);
-            return EOK;
         } else if (error == EPERM) {
             /* case of hidepid=1 mount option for /proc */
             DEBUG(SSSDBG_TRACE_LIBS,
-                  "Proc file [%s] is not permissible, continuing.\n",
+                  "Proc file [%s] is not permissible.\n",
                       path);
-            return EOK;
```

Comment:
returning EPERM was done on purpose see commit a41367f7b6925d9fbb87b8428de4d406e5a50bd1. I think you r commit would regress in this sense?

Reply:
Ah no, you then ignore all error codes, sorry.

Reply:
> returning EPERM

Returning EOK in case of EPERM was done on purpose. But it was done a little bit awkward / buggy, imho.